Initial documentation (#11)
plone/plone.distribution: docs/usage/index.md

> # Using this package

Reviewer: ```suggestion
# Usage
```

> # Using this package

Reviewer: Insert HTML meta stuff

```{toctree}
:maxdepth: 1
:hidden: true

package-structure
code-examples
```

> First of all, a Plone Distribution is a Python Package that can be installed by `pip` and follows certain conventions.

Reviewer: ```suggestion
A Plone distribution is a Python package that can be installed by `pip` and follows certain conventions.
```

> ## Create a new Plone Distribution

Reviewer: ```suggestion
## Create a new Plone distribution
```

> The Plone Community provides a `Cookiecutter` template to generate new Plone Distribution packages: [cookiecutter-plone-distribution](https://github.com/collective/cookiecutter-plone-distribution).

Reviewer: ```suggestion
The Plone community provides a [Cookiecutter](https://www.cookiecutter.io/) template to generate new Plone distribution packages: [`cookiecutter-plone-distribution`](https://github.com/collective/cookiecutter-plone-distribution).
```

> This template is active development and it will probably change its structure until a stable release of `plone.distribution` is made.

Reviewer: ```suggestion
This template is under active development.
It will probably change its structure until a stable release of `plone.distribution` is made.
```

> ### Basic Setup

Reviewer: ```suggestion
### Pre-requisites
```

> To use the template, first, install `Cookiecutter` via command line:

Reviewer: ```suggestion
To use the template, install `Cookiecutter` via the command line.
```

```shell
pip install cookiecutter
```

### Generate the package

> Generate a new Plone Distribution package by running `Cookiecutter`:

Reviewer: ```suggestion
Generate a new Plone distribution package by running the cookiecutter.
```

```shell
cookiecutter gh:collective/cookiecutter-plone-distribution
```

> And answer the questions:

Reviewer: ```suggestion
Answer the questions, replacing the following with your desired options.

```

> ```

Reviewer: ````suggestion
```console
````

distribution_title [Plone Distribution]: Blog
description [Plone Distribution.]: A Blog using the Plone CMS
github_organization [collective]:
python_package_name [collective.blog]: collective.blog
author [Plone Foundation]: Plone Community
email [collective@plone.org]: borg@plone.org
```

> After a while, the folder `collective.blog` will be created, containing the codebase of your Plone Distribution.

Reviewer: ```suggestion
After a while, the folder `collective.blog` (or your chosen name) will be created, containing the code base of your Plone distribution.
```

> ### Install the package (local development)

Reviewer: ```suggestion
### Install the package for local development
```

> Enter the new folder, and install the codebase:

Reviewer: ```suggestion
Enter the new folder, and install the code base.
```

```shell
cd collective.blog
make build
```

> ### Lint and Format the codebase

Reviewer: ```suggestion
### Lint and format the code base
```

> To lint the codebase, use:

Reviewer: ```suggestion
Lint the code base.
```

```shell
make lint
```

> If errors occur, run:

Reviewer: ```suggestion
If errors occur, then run the following command.
```

```shell
make format
```

### Testing

> We recommend the usage of `pytest` and [pytest-plone](https://pypi.org/project/pytest-plone) to test the package.

Reviewer: ```suggestion
We recommend [`pytest`](https://pypi.org/project/pytest/) and [`pytest-plone`](https://pypi.org/project/pytest-plone) to test the package.
```

> The template generator creates a top level folder named `tests` with initial tests. Run them with:

Reviewer: ```suggestion
The template generator creates a top level folder named `tests` with initial tests.
To run the tests, use the following command.
```
